nominees/vaccineledger.json

```diff
@@ -26,15 +34,33 @@
   ],
   "repositories": [
     {
-      "name": "main",
-      "url": "https://gitlab.com/statwig-public"
+      "name": "ledger",
+      "url": "https://gitlab.com/statwig-public/theledger"
+    },
+    {
+      "name": "blockchain engine",
+      "url": "https://gitlab.com/statwig-public/BlockchainEngine"
+    },
+    {
+      "name": "ledger mobile app",
+      "url": "https://gitlab.com/statwig-public/theledgermobileapp"
```

Comment:
The Ledger mobile app appears to be missing a license or any clarification on ownership of intellectual property. Should this be included in the DPG submission, this should be resolved.

Reply:
@jwflory the [LICENSE](https://gitlab.com/statwig-public/theledgermobileapp/-/blob/master/LICENSE) has now been included in the repo. Nice catch!